doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.md

```diff
@@ -0,0 +1,93 @@
+# Sagemaker Distributed Model Parallel 1.1.0 Release Notes
+
+- New Features
+- Bug Fixes
+- Improvements
+- Performance
+- Known Issues
+
+## New Features
+
+The following sections describe new feature releases that are common across frameworks and that are framework specific.
+
+### Common across frameworks
+
+#### Custom slicing support (smp_slice method) for objects passed to smp.step decorated functions
+
+To pass an object to `smp.step` that contains tensors that needs to be split across
+microbatches and is not an instance of list, dict, tuple or set, you should implement `smp_slice` method for the object.
+
+Below is an example of how to use this with PyTorch:
+
+```
+ class CustomType:
+     def __init__(self, tensor):
+         self.data = tensor
+
+     # SMP will call this to invoke slicing on the object passing in total microbatches (num_mb)
+     # and the current microbatch index (mb).
+     def smp_slice(self, num_mb, mb, axis):
+         dim_size = list(self.data.size())[axis]
+
+         split_size = dim_size // num_mb
+         sliced_tensor = self.data.narrow(axis, mb * split_size, split_size)
+         return CustomType(sliced_tensor, self.other)
+
+custom_obj = CustomType(torch.ones(4,))
+
+@smp.step()
+def step(custom_obj):
+    loss = model(custom_obj)
+    model.backward(loss)
+    return loss
+```
+
+### PyTorch
+
+#### Add support for smp.DistributedModel.cpu()
+
+`smp.DistributedModel.cpu()` allgathers parameters and buffers across all `mp_ranks` and moves them to the CPU.
```

Comment:
"allgathers" not sure what this should be

Reply:
You can link to this documentation for allgather : https://sagemaker.readthedocs.io/en/stable/api/training/smd_model_parallel_common_api.html#smp.allgather